index.html

```diff
@@ -8510,7 +8529,7 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                </td>
                <td>Rejuvenation
                </td>
-               <td>An ally within 25ft regains up to 2 resistance points of your choice.
+               <td>An ally within 25ft regains up to 3 resistance points of your choice.
```

Comment:
Could be spicy, that's a total of 9 extra resistance points, so better than Bull's will etc. That could be a bit much even tho I do agree 2 feels pretty meh. Also more flexible as to where they go - tho I guess it's not Self so maybe that's ok.

Reply:
I tend to view it as 5 resistance points is _roughly_ equal to one absorbed damage so around Heavy+MOD.  Considering that we freely give out Medium+MOD healing, I actually think in practice this will be worse than that but due to resistance regen being a bit spicy happy for it to possibly be UP.